plugins/main/ptp/ptp.go

```diff
@@ -83,15 +83,17 @@ func setupContainerVeth(netns ns.NetNS, ifName string, mtu int, pr *current.Resu
 
 		pr.Interfaces = []*current.Interface{hostInterface, containerInterface}
 
-		if err = ipam.ConfigureIface(ifName, pr); err != nil {
-			return err
-		}
-
 		contVeth, err := net.InterfaceByName(ifName)
 		if err != nil {
 			return fmt.Errorf("failed to look up %q: %v", ifName, err)
 		}
 
+		_, _ = sysctl.Sysctl(fmt.Sprintf("net/ipv4/conf/%s/arp_notify", ifName), "1")
```

Comment:
It probably won't hurt anything (Linux should do the "right thing" ), but I don't think we need garp on a point to point link.

Reply:
I can remove it, I was just keeping with the previous behavior.

Reply:
@mccv1r0 I have removed the arp notify setting